bulletroboy/exoforce_simulation.py

```diff
@@ -18,7 +19,8 @@ def __init__(self, cage_conf, human_model, mode):
 
 		self.mode = mode
 		self.operator = Operator(human_model)
-
+		self.operator_publisher = self.create_publisher(PoseStamped, '/roboy/simulation/operator_info', 10)
```

Comment:
this is related to the operator, so it has to be inside of the operator class
look how manuel did it [here](https://github.com/Roboy/bulletroboy/blob/ec8f9510bda694ade19c7837b967ee56c6800c2c/bulletroboy/exoforce.py#L249)

Reply:
The reason why I wrote it there is because exoforce_sim inherits from exoforce which inherits from node so it can create publishers and subscribers with the defined node functions. But the operator class doesnt inherit from it so it is more tricky. I wrote the publish_state function in operator class because (as you said) it does make sense to have it there.
Another option would be to move these "create_publisher" lines of code to operator.py and make the operator class inherit from node. What do you think? :)

Reply:
inherit operator from a node too

Reply:
Done :+1: